Cargo.toml

```diff
@@ -64,9 +64,11 @@ bincode = "1.3.3"
 chrono = "0.4.31"
 derive_builder = "0.12.0"
 
+# memory usage
+jemalloc-ctl = "0.5.4"
+
```

Comment:
You only need this as a dev dependency, no?

Reply:
Yep, I moved it